src/btree/bt_inspect.c

```diff
@@ -0,0 +1,222 @@
+/*-
+ * Copyright (c) 2014-2019 MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include "wt_internal.h"
+
+static int __dump_tree(WT_SESSION_IMPL *, WT_REF *);
+static int __dump_cell_data(WT_SESSION_IMPL *, const char *, const char *, const void *, size_t);
+static int __dump_page_col_var(WT_SESSION_IMPL *, WT_REF *);
+static int __dump_page_row_leaf(WT_SESSION_IMPL *, WT_PAGE *);
+
+/*
+ * __wt_dump --
+ *     Dump the table
+ */
+int
+__wt_dump(WT_SESSION_IMPL *session, const char *cfg[])
+{
+    WT_BTREE *btree;
+    WT_DECL_RET;
+
+    WT_UNUSED(cfg);
+    btree = S2BT(session);
+
+    WT_WITH_PAGE_INDEX(session, ret = __dump_tree(session, &btree->root));
+    return (ret);
+}
+
+/*
+ * __dump_tree --
+ *     Dump the tree, by iterating through tree
+ */
+static int
+__dump_tree(WT_SESSION_IMPL *session, WT_REF *ref)
+{
+    WT_DECL_RET;
+    WT_PAGE *page;
+    const WT_PAGE_HEADER *dsk;
+    WT_REF *child_ref;
+
+    page = ref->page;
+
+    if ((dsk = ref->page->dsk) == NULL) {
+        return (0);
+    }
+    /*
```

Comment:
can you please add white space before this comment.

Reply:
done.